src/rules/noUnsafeQuery.ts

```diff
@@ -34,14 +34,12 @@ const create = (context) => {
         return;
       }
 
-      const { tag } = node.parent;
-
       const tagName =
         node.parent.tag?.name ??
         node.parent.tag?.object?.name ??
         node.parent.tag?.callee?.object?.name;
 
-      const legacyTagName = tag?.name.toLowerCase();
+      const legacyTagName = node.parent.name?.toLowerCase();
```

Comment:
```suggestion
      const legacyTagName = node.parent.tag?.name?.toLowerCase();
```

Reply:
Did this get missed in the merge?

Reply:
@andymakingthings I as remember there are shouldt be `tag` in parent because it in this case parent is not a tag

Reply:
also provided test should prove that it works now

Reply:
I've just confirmed it's no longer giving me an error with the latest code. Thanks!